sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -165,21 +167,67 @@ public AzureMonitorExporterBuilder httpLogOptions(HttpLogOptions httpLogOptions)
      * @return The updated {@link AzureMonitorExporterBuilder} object.
      * @throws NullPointerException If {@code policy} is {@code null}.
      */
-    public AzureMonitorExporterBuilder addHttpPipelinePolicy(HttpPipelinePolicy httpPipelinePolicy) {
+    @Override
+    public AzureMonitorExporterBuilder addPolicy(HttpPipelinePolicy httpPipelinePolicy) {
         if (frozen) {
             throw LOGGER.logExceptionAsError(new IllegalStateException(
-                "httpPipelinePolicies cannot be added after any of the build methods have been called"));
+                "httpPipelinePolicy cannot be added after any of the build methods have been called"));
         }
         httpPipelinePolicies.add(Objects.requireNonNull(httpPipelinePolicy, "'policy' cannot be null."));
         return this;
     }
 
+    /**
+     * Sets the {@link RetryOptions} for all the requests made through the client.
+     *
+     * <p><strong>Note:</strong> It is important to understand the precedence order of the HttpTrait APIs. In
```

Comment:
should we have this warning on the other HttpTrait methods as well?

Reply:
Done